app/views/overrides/organizations/_edit_override.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if @taxonomy.is_a?(Organization) %>
-
-  <%= javascript_include_tag 'katello/organizations/download_certificate' %>
-
+  <% content_for(:javascripts) do -%>
+    <%= javascript_include_tag 'katello/organizations/download_certificate' %>
+  <% end %>
```

Comment:
should we standardize on either
```
<%= javascript … %>
```
OR
```
  <% content_for(:javascripts) do -%>
    <%= javascript_include_tag … %>
  <% end %>
```

using both seems odd?

Reply:
I rather use 
```
  <% content_for(:javascripts) do -%>
    <%= javascript_include_tag … %>
  <% end %>
  ```
  just because when I see `<%= javascript … %>` I assumed it was something default from ruby and not a custom thing

Reply:
I do agree that ths makes it more explicit.